src/main/java/org/skife/jdbi/v2/sqlobject/SqlObjectBuilder.java

```diff
@@ -22,6 +22,7 @@
  */
 public class SqlObjectBuilder
 {
+    static final ParameterBinderRegistry  binderRegistry = new ParameterBinderRegistry();
```

Comment:
It's a problem that SqlObjectBuilder has no where to store data like its registries or default handlers or anything else that might come up as configurable.  DBI could hold the values for components it does not know about that they can retrieve as needed.  Assuming they know about DBI.  I'll look at the chain backwards to see if every path can lead to DBI and come up with something.  As you say elsewhere, "discuss in its own issue".  In v2 flavour I can put into DBI and pollute it a bit more, and then look at the other issue so it can move out of static for v3.